packages/worker/src/tests/TestConfiguration.ts

```diff
@@ -209,7 +210,7 @@ class TestConfiguration {
       sessionId: "sessionid",
       tenantId: user.tenantId,
     }
-    const authCookie = auth.jwt.sign(authToken, coreEnv.JWT_SECRET)
+    const authCookie = jwt.sign(authToken, coreEnv.JWT_SECRET as Secret)
```

Comment:
On a few occasions I have wished for better typing on our environment variables. Might noodle with that one lab day.

Reply:
Yeah agreed, theres definitely more we could do with this.